src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -164,28 +164,27 @@ public void midClimb() {
       McjoystickAxis = -m_climbJoystick.getRawAxis(Constants.leftJoystickY);
       if (McjoystickAxis > 0.1 || McjoystickAxis < -0.1) {
         if (McjoystickAxis > 0) {
-          if (McHeightOne + (McjoystickAxis / 10 * 8) <= Constants.McHeightMax) {
-            McHeightOne = McHeightOne + (McjoystickAxis / 10 * 15);
+          if (McHeightOne + (McjoystickAxis * -1.5) >= Constants.McHeightMin) {
+            McHeightOne = McHeightOne + (McjoystickAxis * -1.5);
           }
-          if (McHeightTwo + (McjoystickAxis / 10 * 8) <= Constants.McHeightMax) {
-            McHeightTwo = McHeightTwo + (McjoystickAxis / 10 * 15);
+          if (McHeightTwo + (McjoystickAxis * -1.5) >= Constants.McHeightMin) {
+            McHeightTwo = McHeightTwo + (McjoystickAxis * -1.5);
           }
         }
         if (McjoystickAxis < 0) {
-          if (McHeightOne + (McjoystickAxis / 10 * 6) >= 0) {
-            McHeightOne = McHeightOne + (McjoystickAxis / 10 * 6);
+          if (McHeightOne + (McjoystickAxis * -0.5) <= Constants.McHeightMax) {
+            McHeightOne = McHeightOne + (McjoystickAxis * -0.4);
           }
-          if (McHeightTwo + (McjoystickAxis / 10 * 6) >= 0) {
-            McHeightTwo = McHeightTwo + (McjoystickAxis / 10 * 6);
+          if (McHeightTwo + (McjoystickAxis * -0.5) <= Constants.McHeightMax) {
+            McHeightTwo = McHeightTwo + (McjoystickAxis * -0.4);
```

Comment:
Make these magic number into constants. Also, we are using only one position value since each side should always match. Remove McHeightTwo